scheduler/core/components/optimizer/dummy.py

```diff
@@ -42,7 +44,7 @@ def _run(self, plans: Plans):
         while not plans.all_done() and len(self.groups) > 0:
 
             ran_group = random.choice(self.groups)
-            if self.add(ran_group, plans):
+            if self.add(ran_group, plans, None):
```

Comment:
Better do:
```def add(self, group: GroupData, plans: Plans, interval: Optional[Interval] = None):```
so we don't need to pass None as a parameter.

Reply:
I was going back and forth trying to decide. Either way works for me, so I'll give it the default parameter.